src/resources/Notifications.js

```diff
@@ -0,0 +1,122 @@
+import Resource from './Resource'
+
+export default class Notifications extends Resource {
+  source = 'notifications';
+
+  /**
+   * List notifications by user
+   *
+   * @param {int} userId - Filter features by userId.
+   * @param {int} start- Starting row for pagination.
+   * @param {int} limit - Limit results per request.
+   * @param {string} [type=''] - Filter results by type.
+   * @param {bool} [unread=''] - Filter results by unread.
+   * @param {bool} [noCache=false] - Flag to turn off the cache.
+   * @return {Object[]} - Notifications.
+   */
+  async list (
+    userId,
```

Comment:
I would do this differently, when you have this many arguments this syntax can be used:

```js
async list ({
  userId,
  start = 0,
  limit = 20,
  type = '',
  unread = '',
  noCache = false
} = {}) {
  // ...
}

// And call like
deskbookers.notifications.list({
  userId,
  start,
  end,
  ...
})
```

This way all arguments are optional / order independent.  Don't forget to update the docs